decrypt_odrive_file.py

```diff
@@ -71,38 +123,41 @@ def decrypt_file(in_file, out_file, password):
     cipher = Crypto.Cipher.AES.new(key, Crypto.Cipher.AES.MODE_CBC, iv)
     next_chunk = ''
     finished = False
-    
     while not finished:
         chunk, next_chunk = next_chunk, cipher.decrypt(in_file.read(4096 * Crypto.Cipher.AES.block_size))
         if len(next_chunk) == 0:
-                chunk = unpad_pkcs7(chunk)
-                fileHash = chunk[-(calcHash.digest_size):]
-                chunk = chunk[:-(calcHash.digest_size)]
-                finished = True
+            chunk = unpad_pkcs7(chunk)
+            fileHash = chunk[-(calcHash.digest_size):]
+            chunk = chunk[:-(calcHash.digest_size)]
+            finished = True
         out_file.write(chunk)
         calcHash.update(chunk)
 
-    print "Original Hash:   {}".format("".join("{:02x}".format(ord(c)) for c in fileHash))
-    print "Calculated Hash: {}".format(calcHash.hexdigest())
+    print("Original Hash:   {}".format("".join("{:02x}".format(ord(c)) for c in fileHash)))
+    print("Calculated Hash: {}".format(calcHash.hexdigest()))
 
 def get_arguments():
     parser = argparse.ArgumentParser()
-    parser.add_argument(u"file", type=str, help=u"The file to decrypt")
-    parser.add_argument(u"password", type=str, help=u"The passphrase")
-    
+    parser.add_argument(u"--path", type=str, help=u"The file to decrypt or the folder to start from",required=True)
```

Comment:
Yeah. A file for the path is assumed to be encrypted. A folder is assumed to have encrypted files inside. If files are encounter that are not encrypted it should error gracefully.